docs/src/man/latex.md

```diff
@@ -21,10 +21,10 @@ This is the binomial coefficient.
 func(x) = # ...
 ```
 
-To avoid needing to escape the special characters the `doc""` string macro can be used:
+To avoid needing to escape the special characters the `@doc raw""` string macro can be used:
```

Comment:
Hmm, technically they are two different macros. Saying something like "`raw""` string macro can be used, but then the docstring needs to be attached with the `@doc` macro" would be a bit more accurate.

Reply:
I've updated the formulation.